packages/vm/test/tester/runners/GeneralStateTestsRunner.ts

```diff
@@ -129,7 +129,7 @@ async function runTestCase(options: any, testData: any, t: tape.Test) {
       const block = makeBlockFromEnv(testData.env, { common })
 
       if (options.jsontrace === true) {
-        vm.evm.events.on('step', stepHandler)
+        vm.evm.events!.on('step', stepHandler)
```

Comment:
Would it make sense in these two instances to check if `vm.evm.events` is defined, and to throw if not?

Reply:
No, in tests we are sure we use default EVM, so we explicitly are sure that we use `EVM` and thus the events are defined :smile:

Reply:
Ok sure!